vendor/BUILD

```diff
@@ -592,11 +592,10 @@ go_library(
         "golang.org/x/net/http2/frame.go",
         "golang.org/x/net/http2/go16.go",
         "golang.org/x/net/http2/go17.go",
-        "golang.org/x/net/http2/go17_not18.go",
+        "golang.org/x/net/http2/go18.go",
```

Comment:
Yeah, that was my reaction. This is why https://github.com/mikedanese/gazel/pull/37 is needed at the same time as we go to 1.8, because this package has strange things going on.